setup.py

```diff
@@ -51,11 +51,12 @@ def find_packages(path, base=""):
     "gevent",
     "mediatimestamp",
     "requests",
-    "nmoscommon"
+    "nmoscommon",
+    "nmos-oauth"
```

Comment:
Is this dependency still required now that the security decorators are held in nmoscommon?